src/mpc-hc/AppSettings.cpp

```diff
@@ -3000,6 +3016,11 @@ bool CAppSettings::CRecentFileListWithMoreInfo::LoadMediaHistoryEntry(CStringW h
             DeserializeHex(dvdPosition, (BYTE*)&r.DVDPosition, sizeof(DVD_POSITION));
         }
     }
+    r.abRepeat.positionA = pApp->GetProfileIntW(subSection, L"abRepeat.positionA", 0) * 10000LL;

```

Comment:
We could skip storing positionAEnabled and positionBEnabled in the settings.
Instead just use:
positionAEnabled = positionA > 0 || positionB > 0;
positionBEnabled = positionB > 0;
For A that would mean an implicit value of 0 could become an explicit value if only B was set initially, but since that doesn't impact the actual behavior, I think it is a worthwhile optimization.

Reply:
> We could skip storing positionAEnabled and positionBEnabled in the settings.
> 
> Instead just use:
> 
> positionAEnabled = positionA > 0 || positionB > 0;
> 
> positionBEnabled = positionB > 0;
> 
> For A that would mean an implicit value of 0 could become an explicit value if only B was set initially, but since that doesn't impact the actual behavior, I think it is a worthwhile optimization.

I think it's true. Guess we could eliminate the booleans entirely in the code.